### PR TITLE
Change repo from sdk to install-scripts in scripts

### PR DESCRIPTION
**Pull request by AR-May**
Delivers #38.  Change repo from sdk to install-scripts in scripts' exception messages.

### CODE REVIEW

**Comment by AR-May** [comment]
I have found only one message with a link to https://github.com/dotnet/sdk/issues which should be replaced.

**Review by bozturkMSFT — APPROVED** [review]

File: src/dotnet-install.ps1 [hunk 1]
```diff
@@ -195,7 +195,7 @@ function Get-CLIArchitecture-From-Architecture([string]$Architecture) {
         { $_ -eq "x86" } { return "x86" }
         { $_ -eq "arm" } { return "arm" }
         { $_ -eq "arm64" } { return "arm64" }
-        default { throw "Architecture not supported. If you think this is a bug, report it at https://github.com/dotnet/sdk/issues" }
+        default { throw "Architecture not supported. If you think this is a bug, report it at https://github.com/dotnet/install-scripts/issues" }
```
Comment (donJoseLuis) [inline src/dotnet-install.ps1:198]: would it add value to include the specified architecture which is not supported?

@AR-May  I know, not your change...while we're here, would the above make our exception better?

@bozturkMSFT , what do you think?

**Review by donJoseLuis — APPROVED** [review]
minor suggestion

File: src/dotnet-install.ps1 (hunk 1 above)
Comment (AR-May) [reply src/dotnet-install.ps1:198]: For me, it sounds reasonable. I can edit the message like that: "Architecture '$Architecture' not supported.".
@bozturkMSFT, what do you think of this?

**Review by grinrag — APPROVED** [review]

File: src/dotnet-install.ps1 (hunk 1 above)
Comment (bozturkMSFT) [reply src/dotnet-install.ps1:198]: Not a "must", in my opinion, but it would be nice to have.

**Pull request merged**